### PR TITLE
Change forward msg

### LINKED ISSUE #2642: When you forward an email, the feedback message is wrong
Its says Reply Sent! Instead should be The email was sent or another message

### PR DESCRIPTION
**Pull request by GretaD**
I think new message sent, reply message and forward message should have the same feedback message

Fixes #2642

### CODE REVIEW

File: src/components/Composer.vue [hunk 1]
```diff
@@ -174,7 +174,7 @@
 	</div>
 	<div v-else class="emptycontent">
 		<h2 v-if="!isReply">{{ t('mail', 'Message sent!') }}</h2>
-		<h2 v-else>{{ t('mail', 'Reply sent!') }}</h2>
+		<h2 v-else>{{ t('mail', 'Message sent!') }}</h2>
```
Comment (ChristophWurst) [inline src/components/Composer.vue]: sure, but in that case we don't need the if-else, do we? :)

Comment (GretaD) [reply src/components/Composer.vue]: We need one for new message v-if, and one for everything else, v-else.

Comment (ChristophWurst) [reply src/components/Composer.vue]: Look again. It's the same element in both cases: `<h2>{{ t('mail', 'Message sent!') }}</h2>`

Comment (GretaD) [reply src/components/Composer.vue]: Please check isReply function. If we remove v-else, reply and forward will not have a feedback message.

Comment (ChristophWurst) [reply src/components/Composer.vue]: Let me rephrase. Instead of

```html
<h2 v-if="!isReply">{{ t('mail', 'Message sent!') }}</h2>
<h2 v-else>{{ t('mail', 'Message sent!') }}</h2>
```

we can simply do

```html
<h2>{{ t('mail', 'Message sent!') }}</h2>
```

`v-if` and `v-else` render the same element.

Comment (GretaD) [reply src/components/Composer.vue]: dammit
